ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_additional.robot

```diff
@@ -116,8 +116,6 @@ Verify User Can Remove GPUs From Workbench
     ...    pv_description=${EMPTY}  pv_size=${PV_SIZE}    gpus=1
     Run Keyword And Continue On Failure    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_TITLE_GPU}
     Sleep    10s     reason=There is some delay in updating the GPU availability in Dashboard
-    Run Keyword And Continue On Failure    GPU Dropdown Should Be Disabled    workbench_title=${WORKBENCH_TITLE_GPU}
-    Click Button    ${GENERIC_CANCEL_BTN_XP}
```

Comment:
why removing these lines?

Reply:
Because this is not a dropdown anymore, is a text field that can be modified

Reply:
does the field gets disabled when no accelerator is available?